JASP-Engine/JASP/R/regressionlogistic.R

```diff
@@ -599,60 +596,122 @@ RegressionLogistic <- function(jaspResults, dataset = NULL, options, ...) {
+                                              R = options$coeffEstimatesBootstrappingReplicates,
+                                              model.formula = formula(glmObj[[i]]),
+                                              options = options,
+                                              envir = envir),
+                                   silent = TRUE)
+          bootstrap.summary$stdEst   <- envir$stdEst
+          bootstrap.summary$robustSE <- envir$robustSE
+          bootstrapResults[[bootname]] <- bootstrap.summary
+          try(rm(envir, envir = parent.env(envir)), silent = TRUE)
+
+        } else {
+          bootstrap.summary <- bootstrapResults[[bootname]]
+        }
+
       bootstrap.coef <- matrixStats::colMedians(bootstrap.summary$t, na.rm = TRUE)
+      bootstrap.std  <- matrixStats::colMedians(bootstrap.summary$stdEst, na.rm = TRUE)
       bootstrap.bias <- colMeans(bootstrap.summary$t, na.rm = TRUE) - bootstrap.summary$t0
-      bootstrap.se   <- matrixStats::colSds(as.matrix(bootstrap.summary$t), na.rm = TRUE)
+      bootstrap.or   <- matrixStats::colMedians(exp(bootstrap.summary$t), na.rm = TRUE)
+      if (options$robustSEOpt)
+        bootstrap.se <- matrixStats::colMedians(bootstrap.summary$robustSE)
+      else
+        bootstrap.se <- matrixStats::colSds(as.matrix(bootstrap.summary$t), na.rm = TRUE)
       
       for (j in seq_along(rn)) {
-        result.bootstrap.ci <- try(boot::boot.ci(bootstrap.summary, type = "bca", conf = 0.95, index=j))
-        if(!isTryError(result.bootstrap.ci))
-          bootstrap.ci <- result.bootstrap.ci
-        else if(identical(attr(result.bootstrap.ci, "condition")$message, "estimated adjustment 'a' is NA")){ #Does this still work if we turn on translations? I'm gonna guess not...
-          ci.fails <- TRUE
-          bootstrap.ci <- list(bca = rep(NA, 5))
-        } else
-          bootstrap.ci <- result.bootstrap.ci
         
-        if(ci.fails)
-          estimatesTableBootstrap$addFootnote(gettext("Some confidence intervals could not be computed.\nPossibly too few bootstrap replicates."), symbol = gettext("<i>Note.</i>"))
-
         row <- list(
           model = as.character(i),
           param = .formatTerm(rn[j], glmObj[[i]]),
           est   = as.numeric(bootstrap.coef[j]),
           bias  = as.numeric(bootstrap.bias[j]),
           se    = as.numeric(bootstrap.se[j]),
-          cilo  = as.numeric(bootstrap.ci$bca[4]),
-          ciup  = as.numeric(bootstrap.ci$bca[5]),
           .isNewGroup = as.logical(j == 1)
         )
 
+        if (options$coeffCI) {
+          result.bootstrap.ci <- try(boot::boot.ci(bootstrap.summary, type = "bca", conf = options$coeffCIInterval, index=j))
+          if(!isTryError(result.bootstrap.ci))
+            bootstrap.ci <- result.bootstrap.ci
+          else if(identical(attr(result.bootstrap.ci, "condition")$message, "estimated adjustment 'a' is NA")){ #Does this still work if we turn on translations? I'm gonna guess not...
```

Comment:
It'll only stop working if the package uses `gettext()` and includes a .po file for the language.. I suppose we made error message checking even more fragile ;)